store/AuthStore.ts

```diff
@@ -0,0 +1,68 @@
+import { login } from '@/actions/login'
+import { logout } from '@/actions/logout'
+import { create } from 'zustand'
+import { persist } from 'zustand/middleware'
+
+interface AuthState {
+  user: any | null
```

Comment:
Type can be `User | null`. `User` type can be imported from supabase library

Reply:
Got it, I'll make the changes right away.